inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.cpp

```diff
@@ -3,10 +3,11 @@
 //
 
 #include "layer_test_utils.hpp"
+#include "cpp_interfaces/interface/ie_internal_plugin_config.hpp"
 
 namespace LayerTestsUtils {
 
-LayerTestsCommon::LayerTestsCommon() : threshold(1e-2f) {
+LayerTestsCommon::LayerTestsCommon() : threshold(1e-2f), lpMode(LpMode::DEFAULT) {
```

Comment:
If you need such functionality I request to subclass `LPTTestCommon` from `LayerTestsCommon` since `lpMode` is not required in any single layer tests for which this class was created initially. 

If we will try to fit all of required functionality in one class it will be pretty hard to maintain in the future

Reply:
Correct, we have the subclass already: `LayerTransformation`. But `lpMode` is used in not configurable `LayerTestsCommon::LoadNetwork()` methid. If I move `lpMode` to `LayerTransformation` subclass I should override network loading functionality. We will have second network loading implementation.   
As variant, I can add `config`  input argument in `LayerTestsCommon::LoadNetwork` to have configurable implementation in base class which will be reused from `LayerTransformation`. Is it OK for you?

Reply:
`LayerTestsCommon` has a method `ConfigurePlugin` which is called before `LoadNetwork`. You can override it and add the code from https://github.com/openvinotoolkit/openvino/pull/799/files#diff-c53835e3edab45558c9168e909684f2aR88

Reply:
But don't forget to take into account, that config can be defined externally. So you have to update class attribute `configгation` instead of overriding it

Reply:
unfortunately configuration with `core->SetConfig` (from `LayerTestsCommon::ConfigurePlugin()`) is different from configuration with 'core->LoadNetwork(cnnNetwork, targetDevice, config)' (from `LayerTestsCommon::LoadNetwork`): options defined by core->SetConfig fill another map. The variant with virtual method is used, please review.